src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -575,7 +575,7 @@ protected virtual void VisitLvalue(BoundLocal node)
         /// Visit a boolean condition expression.
         /// </summary>
         /// <param name="node"></param>
-        protected void VisitCondition(BoundExpression node)
+        protected virtual void VisitCondition(BoundExpression node, bool inExpression = false)
```

Comment:
>false [](start = 88, length = 5)

This parameter probably shouldn't have default value.

Reply:
Removed parameter.

---
In reply to: [155053173](https://github.com/dotnet/roslyn/pull/23439#discussion_r155053173) [](ancestors = 155053173)